keras_nlp/utils/text_generation_test.py

```diff
@@ -14,9 +14,11 @@
 """Tests for Text Generation Utils."""
 
 
+import numpy as np
 import tensorflow as tf
 
 from keras_nlp.utils.text_generation import greedy_search
+from keras_nlp.utils.text_generation import random_sampling
 
 
 class TextGenerationTest(tf.test.TestCase):
```

Comment:
We should split a separate class for greedy, random, etc. They should mirror each other, but not be mixed in the same unit tests.